readalongs/cli.py

```diff
@@ -66,116 +73,132 @@ def cli():
     """Management script for Read Along Studio."""
 
 
-@app.cli.command(context_settings=CONTEXT_SETTINGS, short_help='Force align a text and a sound file.')
-@click.argument('inputfile', type=click.Path(exists=True, readable=True))
-@click.argument('wavfile', type=click.Path(exists=True, readable=True))
-@click.argument('output-base', type=click.Path())
-@click.option('-b', '--bare', is_flag=True, help='Bare alignments do not split silences between words')
-@click.option('-c', '--closed-captioning', is_flag=True, help='Export sentences to WebVTT and SRT files')
-@click.option('-d', '--debug', is_flag=True, help='Add debugging messages to logger')
-@click.option('-f', '--force-overwrite', is_flag=True, help='Force overwrite output files')
-@click.option('-i', '--text-input', is_flag=True, help='Input is plain text (assume paragraphs separated by blank lines, 1 paragraph per page)')
-@click.option('-l', '--language', type=click.Choice(LANGS, case_sensitive=False), help='Set language for plain text input')
-@click.option('-u', '--unit', type=click.Choice(["w", "m"], case_sensitive=False), help='Unit (w = word, m = morpheme) to align to')
-@click.option('-s', '--save-temps', is_flag=True,
-              help='Save intermediate stages of processing and temporary files (dictionary, FSG, tokenization etc)')
-@click.option('-t', '--text-grid', is_flag=True, help='Export to Praat TextGrid & ELAN eaf file')
-@click.option('-x', '--output-xhtml', is_flag=True, help='Output simple XHTML instead of XML')
+@app.cli.command(context_settings=CONTEXT_SETTINGS, short_help="Force align a text and a sound file.")
+@click.argument("inputfile", type=click.Path(exists=True, readable=True))
+@click.argument("audiofile", type=click.Path(exists=True, readable=True))
+@click.argument("output-base", type=click.Path())
+@click.option("-b", "--bare", is_flag=True, help="Bare alignments do not split silences between words")
+@click.option("-c", "--config", type=click.Path(exists=True), help="Use ReadAlong-Studio configuration file")
```

Comment:
Add `(in JSON format)` at the end of the `help` string.
```suggestion
@click.option("-c", "--config", type=click.Path(exists=True), help="Use ReadAlong-Studio configuration file (in JSON format)")
```